test/unit/Divider.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import Divider from 'src/divider';
+
+describe('<Divider />', () => {
+  it('renders className', () => {
+    const wrapper = shallow(
+      <Divider
+        className="test-class-name"
+      />
+    );
+
+    assert.ok(wrapper.is('.test-class-name'), 'should contain the className');
+  });
+
+  it('renders inset', () => {
+    const wrapper = shallow(
+      <Divider
+        inset={true}
+      />
+    );
+
+    assert.equal(wrapper.html(),
+      '<hr style="margin:0;margin-top:-1px;margin-left:72px;height:1px;border:none;background-color:#e0e0e0;"/>',
```

Comment:
@alitaheri @pradel Right now the shallow wrapper represents `muiThemeable()` not `Divider`.

This makes things confusing when looking at the shallow renderer output. Now, if I call `.render()` on the wrapper to create a `Cheerio` object and grab the `children()` (which has the actual divider, the root is the wrapper)

```js
const cheerioDivider = wrapper.render().children();
```

Now I have access to this:
```
{ type: 'tag',
     name: 'hr',
     attribs: { style: 'margin:0;margin-top:-1px;margin-left:72px;height:1px;border:none;background-color:#e0e0e0;' },
     children: [],
     next: null,
     prev: null,
     parent: null,
     root:
      { type: 'root',
        name: 'root',
        attribs: {},
        children: [Object],
        next: null,
        prev: null,
        parent: null } },
```

You can either

a) check for a string match in the attrs as seen there
b) use cheerio to get the style property!!

```js
console.log(wrapper.render().children().css('margin-left'));

// logs "72px"
```